### ISSUE TITLE
Control over whitespace after components

### ISSUE THREAD

**Issue by doughsay** [petalframework/petal_components#10]
This is sort of a general HTML problem I face all the time, but is exacerbated by component libraries like this and I was wondering if there is an elegant way to fix this.

Imagine you want to render a comma separated list of links (<a> tags). So, simplified:

```html
<a>foo</a>, <a>bar</a>, <a>baz</a>
```

![DeepinScreenshot_select-area_20220214143237](https://user-images.githubusercontent.com/221693/153957815-c1587cff-476b-4c59-b679-f417da1c5d1b.png)

This works, because you can directly put the comma after the tag, without any whitespace.  But if you're using a component library (such as this one), the link component usually ends up outputting extra whitespace or a newline after the component. Example:

```html
<.link to="/" label="foo" />, <.link to="/" label="bar" />, <.link to="/" label="baz" />
```

![DeepinScreenshot_select-area_20220214143254](https://user-images.githubusercontent.com/221693/153957826-8b2f193a-82c0-418e-b571-140c0a93d3b3.png)

This doesn't render correctly, there's a space between the end of the link and the comma.

The only recourse I generally have is to just manually construct the HTML, which sticks out like a sore thumb. Is it possible to remove the trailing space/newline from the link component?

**Comment by mplatts** [comment]
Hi @doughsay ,

Thanks for this - I pushed a fix - can you try out version 0.10.8.

Code: https://github.com/petalframework/petal_components/commit/1e5c21b84094de99cb119cf369426b75e8a871af

**Comment by doughsay** [comment]
Beautiful!  And so fast!  Thanks!  Works perfectly!
